components/RecentProjectActivity.tsx

```diff
@@ -11,7 +11,7 @@ type RecentProjectActivityProps = {
 const RecentProjectActivity = ({ projects }: RecentProjectActivityProps): JSX.Element => {
 	return (
 		<Card sx={styles.card} elevation={2}>
```

Comment:
Styles to match that of the 'How It Works' section, copied from `sectionHeading` and `howItWorksText`: 

```js
const styles = {
  header: {
    mb: 5,
		fontSize: '3rem',
		'@media (min-width: 600px)': {
			fontSize: '4rem',
		},
  },
  subheader: {
    fontSize: '1.5rem',
	  mb: 7,
	  '@media (min-width: 600px)': {
		  fontSize: '1.75rem',
		  mb: 10,
	  },
	}
}
```

The component JSX should be refactored to be as so:

```xml
return (
  <section>
    <Typography variant="h2" sx={styles.header}>Recent Activity</Typography>
    <Typography sx={styles.subheader}>Check out a few of the most recently updated projects on Arbor.</Typography>
    <Grid container spacing={3}>
			{projects.map(project => (
				<Grid item xs={12} sm={4} key={project._id}>
					<ProjectCard details={project} />
				</Grid>
			))}
		</Grid>			
  </section>
)
```

Or if you're importing the styles to this file, then you can just reuse those variables as the styles in `sx`.